examples/simple-game-server/main.go

```diff
@@ -807,8 +820,12 @@ func getListValues(s *sdk.SDK, listName string) (string, error) {
 	values, err := s.Alpha().GetListValues(listName)
 	if err != nil {
 		log.Printf("Error getting List %s values: %s", listName, err)
+		return strings.Join(values, ","), err
 	}
-	return strings.Join(values, ",") + "\n", err
+	if len(values) > 0 {
+        return "VALUES: " + strings.Join(values, ",") + "\n", nil
+    }
+	return "Empty List" + strings.Join(values, ",") + "\n", err
```

Comment:
Could you review the getListValues method? I'm unsure if anything is missing. In negative test cases, it only returns 'ERROR:' without further details. Is this behavior expected?

Reply:
It's normal for the examples/simply-game-server/main.go code. This code only sends back the return value, not any return error. And in the case of `GetListValues` the return value is nil. We probably should pipe through the error message, but that's not in the code as it stands. 

https://github.com/googleforgames/agones/blob/12159252599bf2407cc3f666796a39a9ce4b8eb3/examples/simple-game-server/main.go#L496-L511 is where it only returns the response.